src/content/docs/change-tracking/change-tracking-webhooks.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Notify your team about deployments'
+tags:
+  - APM
+metaDescription: "After you set up the changes you want to monitor, you can use a webhook to notify your colleagues about the impacts of those changes."
+---
+
+
+import webHookTest from 'images/tracking_screenshot-crop_webhook_test.webp'
+
+
+
+After you record a deployment for an APM application entity, whether it's with the change tracking feature or the older [REST API](/docs/apm/apm-ui-pages/events/record-deployments/#api-instructions), you can notify your team members using a webhook.
```

Comment:
This feels like a long intro sentence. Is it possible to break it up into two sentences? Also, why would someone want to notify their team using a webhook? What's the benefit when it comes to change tracking. Is there a value prop that's needed?

Reply:
@homelessbirds I could break this up like this: "After you record a deployment for an APM application entity, you can keep your team informed about those changes through the use of a webhook. Webhooks are available whether you record a deployment using the tracking feature or the older [REST API](/docs/apm/apm-ui-pages/events/record-deployments/#api-instructions).

Reply:
I like this @rhetoric101 - I think it's a strong intro.